challenges/01 Goldilocks' Bear Necessities/app.js

```diff
@@ -1 +1,96 @@
-// Do awesome stuff...
+ let fs = require('fs') //require fs library
+
+
+ /**
+  * Gets suitable chairs array from file with weight and temperature pairs
+  * @param  string file Filepath
+  */
+ function getSuitableChairs(file) {
+     let content = fs.readFileSync(file, 'utf8')
+     if (content) {
+         let lines = content.split('\n') //split content by each new line
+         let goldislocksProps = getGoldilocksWeightAndTemperature(lines)
+         let chairs = getChairsWeightAndTemperature(lines)
+         let suitableChairs = findSuitableChairsForGoldilocks(goldislocksProps, chairs)
+         return suitableChairs
+     } else
+         return null
+ }
+
+ /**
+  * Parse Goldilocks and chair's weight and temperature from each line and save it in in properties this.weight and this.maxTemperature or in an array this.chairs
+  * @param  array lines Lines array from file
+  * @return {object Object with Goldilock's weight and temperature
+  */
+ function getGoldilocksWeightAndTemperature(lines) {
+     let weight = parseInt(lines[0].split(' ')[0]) //get weight of Goldilocks from first line
+     let maxTemperature = parseInt(lines[0].split(' ')[1]) //get maximum temperature that Goldilocks can tolerate from first line
+     return { weight: weight, temperature: maxTemperature }
+ }
+
+ /**
+  * Parse Goldilocks and chair's weight and temperature from content string and save it in in properties this.weight and this.maxTemperature or in an array this.chairs
+  * @param  string content Content string returned from fs.readFile
+  * @return array chairs Chairs array with weight and temperature
```

Comment:
Hier könnte man mehr spezifizieren, wie das Array mit den Stühlen aussieht, sodass aus dem Kommentar klar wird, dass es ein Array von Objekten mit den Eigenschaften _weight_ und _temperature_ ist.
Hinweis: In Kommentaren ist es erlaubt Arrays folgendermaßen zu definieren: `@return Object[] Array mit Objekten...`.